.gitignore

```diff
@@ -1,3 +1,6 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+
+docs/build
+docs/site
```

Comment:
Note the .gitignore in the docs directory.

Reply:
So this is not necessary, right?

Reply:
Thanks I'll remove this.  Strangely I distinctly remember checking for this... I must have been in the working directory.